capa/features/extractors/binexport2/__init__.py

```diff
@@ -259,15 +260,15 @@ def _find_base_address(self):
 
     def _compute_thunks(self):
         for addr, idx in self.idx.vertex_index_by_address.items():
-            vertex = self.be2.call_graph.vertex[idx]
+            vertex: BinExport2.CallGraph.Vertex = self.be2.call_graph.vertex[idx]
```

Comment:
we could optionally import these nested names directly to avoid having to namespace them (which is a little noisy imho).